deploy/upgrade/config.ts

```diff
@@ -0,0 +1,100 @@
+import fs from 'fs';
+import path from 'path';
+import { isAddress, Provider } from 'ethers';
+import {
+  IDelegationManager__factory,
+  IPool__factory,
+  IStrategy__factory,
+  IStrategyManager__factory,
+} from '../../typechain-types';
+import { assertTokenConfig, EthConnectionConfig, TokenConfig } from '../config-common';
+
+export interface UpgradeConfig {
+  ethConnection: EthConnectionConfig;
+  aaveYield?: AaveYieldUpgradeConfig;
+  ethYield?: EthYieldUpgradeConfig;
+}
+
+export interface AaveYieldUpgradeConfig {
+  underlyingToken: string;
+  wardenHandler: WardenHandlerConfig;
+}
+
+export interface EthYieldUpgradeConfig {
+  lidoWithdrawalQueue: string;
+  wardenHandler: WardenHandlerConfig;
+}
+
+export interface WardenHandlerConfig {
+  axelarGateway: string;
+  axelarGasService: string;
+  wardenChain: string;
+  wardenContractAddress: string;
+}
+
+export const configAllowedKeys = new Set<string>(['ethConnection', 'aaveYield', 'ethYield']);
+
+export async function loadUpgradeConfig(network: string, provider: Provider, dryRun: boolean): Promise<UpgradeConfig> {
+  const configDir = path.join(path.parse(__dirname).dir, `data`, `configs`, network);
+
+  if (!fs.existsSync(configDir)) {
+    throw new Error(`Directory '${configDir}' does not exist`);
+  }
+  if (!fs.statSync(configDir).isDirectory()) {
+    throw new Error(`Specified '${configDir}' is not a directory`);
+  }
+  const configFilename = path.join(configDir, 'config.json');
```

Comment:
Will it conflict with the deploy config? I think that we should rename config for upgrades, something like: `config-upgrade-v1-to-v2.json`